package.json

```diff
@@ -2,6 +2,7 @@
   "name": "cn.ubuntu.com",
   "description": "Django website for cn.ubuntu.com",
   "scripts": {
+    "start": "yarn run build && yarn run serve",
```

Comment:
Typically we make use of `concurrently` in the `start` script, so that we're setting up watchers at the same time as running the server. [See e.g. ubuntu.com](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/package.json#L19):

``` json
{
  "scripts": {
    // ...
    "watch-css": "watch -p 'static/sass/**/*.scss' -p 'node_modules/vanilla-framework/scss/**/*.scss' -c 'yarn run build-css'",
    "watch-js": "watch -p 'static/js/src/**/*.js' -p 'static/js/data/**/*.js' -p 'static/js/third-party/**/*.js' -c 'yarn run build-js'",
    "start": "yarn run build && concurrently --raw 'yarn run watch-css' 'yarn run watch-js' 'yarn run serve'",
    // ...
  }
  "devDependencies": {
    // ...
    "concurrently": "5.2.0",
    // ...
  }
}
```